index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Microverse Portfolio Website</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <nav id="mobile-navbar">
+            <img src="assets/menu.png" alt="">
+        </nav>
+    </header>
+    <section id="headline-section">
+        <div class="container">
+            <h1 >
+                <span id="name-heading">
+                    Hey there<br>
+                I'm Abdul Salam
+                </span>
+                <span id="job-role">
+                    I',m a Software<br>
+                    Developer
+                </span>
```

Comment:
- [ ] The main title of the headline section should be wrapped in an `h1` tag for proper HTML page structure reasons, Kindly fix this.

Reply:
Hi you can see the code it is already wrapped with h1 tag but due to different color of text in heading I use span to apply different colors and styles